### PR TITLE
DEMO-231: Embed form improvements

### PR DESCRIPTION
**Pull request by ciastektk**
https://jira.ez.no/browse/DEMO-231

ezplatform ee demo local form

### CODE REVIEW

File: app/Resources/views/themes/tastefulplanet/full/form.html.twig [hunk 1]
```diff
@@ -5,16 +5,14 @@
         <div class="container">
             <div class="row">
                 <div class="col-md-8 offset-md-2">
-                    <div class="field-description">
+                    <div class="block-header__name">
```
Comment (sunpietro) [inline app/Resources/views/themes/tastefulplanet/full/form.html.twig]: I see you're trying to implement BEM naming convention. I'm missing the context of a `Block`. What you did here is adding a definition of `Element` from BEM.
This is incorrectly implemented code.

**Comment by ciastektk** [comment]
@sunpietro can you review once again?

File: app/Resources/views/themes/tastefulplanet/form_builder/default_form_view.html.twig [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<section class="content-type-form full-view">
```
Comment (sunpietro) [inline app/Resources/views/themes/tastefulplanet/form_builder/default_form_view.html.twig]: Maybe `demo-form`? I believe you should consider using BEM naming convention consequently throughout all you code

Comment (damianz5) [reply app/Resources/views/themes/tastefulplanet/form_builder/default_form_view.html.twig]: the name is consistent with the rest of the files (folder, product, place, article etc) on full and embeded view. Changing it to BEM will be great, we will create a separate PR for that, as global change on demo templates

Comment (damianz5) [reply app/Resources/views/themes/tastefulplanet/form_builder/default_form_view.html.twig]: We will improve it as a separate issue https://jira.ez.no/browse/DEMO-237

**Pull request merged**